std_msgs/Header header
uint32           id                 # Label der ConnectedComponent
geometry_msgs/Point centroid_pixel   # Pixelkoordinaten der Ampel in Semantic-Aufl√∂sung (x,y als Spalten-/Zeilenindex)
float32          distance           # Median-Distanz in Metern

